_docs/data-workflow.md

```diff
@@ -15,16 +15,16 @@ One function of the Glider DAC is to coordinate with the NWS' National Data Buoy
 
 ### NDBC Glider Page
 
-Active and historical glider deployments are posted to an [NDBC glider page](http://www.ndbc.noaa.gov/gliders.php).  These questions and answers from NDBC clarify the information on the glider page specific information and GTS.  
+Active and historical glider deployments are posted to an [NDBC glider page](https://www.ndbc.noaa.gov/gliders.php).  These questions and answers from NDBC clarify the information on the glider page specific information and GTS.  
 
  1. **What constitutes "active?"  Right now, 11 gliders are in the active column, but are they all going to GTS or what makes them active?**
  If a glider is shown as active, it is presently deployed and making data available to NDBC for processing.  This does not guarantee that data are released to the GTS.  Data may post on the website even if it does not meet requirements for release to the GTS.  Right now all gliders shown as active are being released to the GTS
- 2. **Do they transition to historical as soon as they are recovered or is there a different mechanism to transition them?**             
+ 1. **Do they transition to historical as soon as they are recovered or is there a different mechanism to transition them?**             
  A glider is no longer be classified as active if new data has not been available for five days.  This is automatically done by the web processing software.
 
 ### The Observing System Monitoring Center (OSMC)
 
-[OSMC](www.osmc.info) is a monitoring tool that attempts to harvest and track all ocean observations on the GTS.  This tool is comprehensive but not perfect so if discrepancies are found we encourage users to contact OSMC directly OSMC.Webmaster-AT-noaa-DOT-gov.
+[OSMC](https://www.osmc.noaa.gov/) is a monitoring tool that attempts to harvest and track all ocean observations on the GTS.  This tool is comprehensive but not perfect so if discrepancies are found we encourage users to contact OSMC directly OSMC.Webmaster-AT-noaa-DOT-gov.
```

Comment:
@kbailey-noaa some links, like this one, I'm not 100% sure the new one is the right substitute for the dead one.

Reply:
@ocefpaf the content in this .md file is obsolete (e.g. NDBC doesn't publish glider data anymore) and I don't think this .md is even linked in the sidebar/published to https://ioos.github.io/glider-dac/.  Recommend removing (or similar action to glider-dac-communications-plan.md).

Reply:
Done!